### DISCUSSION TITLE
About the Darcy variable problem, and a question about derivative of function spaces (a bug?)

### DISCUSSION THREAD

**Discussion by BraveDrXuTF** [lululxvi/deepxde#1572] [Q&A]
In pi-deeponet, I'm considering the darcy problem $\nabla \cdot k(\mathbf{x}) \nabla T=Q $, where k is sampled from a Gaussian process. Now I have a question. I notice the function space provides its sampling with numpy package, which means it does not consider the gradient backward. in the darcy equation, as you can see, $k(x)$ is a function with spatial variable, so we have to derive its derivatives. But it seems impossible to do that in the package.

**Comment by lululxvi** [comment]
You are right. This case is not supported yet.
